src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    private boolean isDone;
+    private final String taskName;
```

Comment:
You could look into having "taskname" capitalized as it has been declared as final which would imply that it not a variable. 

````suggestion
    private final String TASK_NAME;
````